app/js/main.js

```diff
@@ -172,6 +172,16 @@ function setupUpdateSubscription(subscriptions) {
   );
 }
 
+const EntitlementType = {
```

Comment:
EntitlementType is pretty broad term. What about something like "ShowcaseEntitlementEvent"?

Reply:
A comment could help improve clarity too

Reply:
Could you add a comment too?